mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,620 @@
+        filename="da-tacos_benchmark_subset_madmom.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_madmom.zip?download=1",
+        checksum="8beb1d8fa39f95b79d5f502a41fd5f0c",
+        destination_dir=".",
+    ),
+    "benchmark_mfcc": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_mfcc.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_mfcc.zip?download=1",
+        checksum="a3be0cd80754043a8c238cf501062789",
+        destination_dir=".",
+    ),
+    "coveranalysis_tags": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_tags.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_tags.zip?download=1",
+        checksum="4b9d4cd5beca571e1d614c9a77580f8c",
+        destination_dir=".",
+    ),
+    "coveranalysis_cens": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_cens.zip",
+        url="https://zenodo.org/record/4717628/files/da-tacos_coveranalysis_subset_cens.zip?download=1",
+        checksum="7eb56dd3a44fa7d90cc6643bc446e79b",
+        destination_dir=".",
+    ),
+    "coveranalysis_crema": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_crema.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_crema.zip?download=1",
+        checksum="70252fe115e1ab4c4d74698d4ad68f4b",
+        destination_dir=".",
+    ),
+    "coveranalysis_hpcp": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_hpcp.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_hpcp.zip?download=1",
+        checksum="961784fc2419214adf05504e9fc56cc2",
+        destination_dir=".",
+    ),
+    "coveranalysis_key": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_key.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_key.zip?download=1",
+        checksum="6e72db855bad5805a67382bd318eee9c",
+        destination_dir=".",
+    ),
+    "coveranalysis_madmom": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_madmom.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_madmom.zip?download=1",
+        checksum="42482eedfe9d9a8be9db3611b9d343b4",
+        destination_dir=".",
+    ),
+    "coveranalysis_mfcc": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_mfcc.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_mfcc.zip?download=1",
+        checksum="11371910cad7012daaa81a5fe9dfa1c0",
+        destination_dir=".",
+    ),
+}
+INDEXES = {
+    "default": "1.1_full",
+    "test": "1.1_full",
+    "crema": "1.1_crema",
+    "1.1_crema": core.Index(filename="da_tacos_index_1.1_crema.json"),
```

Comment:
you can specify partial downloads here, if it's different from the full list of keys. If someone selects this dataset version, it will by default download everything in the specified list below. If partial_downloads=None, it should match the full list of keys in REMOTES
```suggestion
    "1.1_crema": core.Index(filename="da_tacos_index_1.1_crema.json", partial_download=["benchmark_crema", "coveranalysis_crema"]),
```